app/views/posts/new.html.erb

```diff
@@ -0,0 +1,12 @@
+<h5> Create a new Post:</h5>
+<%= form_with model: @post, url: user_posts_path do |form| %>
+   <div class="form-group">
+    <%= form.label :title , "Title" %>
+    <%= form.text_field :title %>
+   </div>
+   <div class="form-group">
+    <%= form.label :text , "Text" %>
+    <%= form.text_area :text , size: '40*40'%>
+   </div> 
+    <%= form.submit "Post" %>
+<% end %>
```

Comment:
- [ ] Nice adding the form to create the post however it doesn't create it because of `validation of likes_counter and comments_counter` because they are nil at the moment when creating the post so the creation process roll back I suggest adding a default value  `0` to likes_counter and comments counter so that. the post will be created without problem 
- you can add default value to a column with : 
```ruby
change_column :table_name, :column, :type, default: 0
```